src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/ListingMapper.java

```diff
@@ -59,6 +61,41 @@ public Listing getListing(HearingDetails hearingDetails, List<String> preferredH
             listing.setListingJohTiers(new ArrayList<>(hearingDetails.getPanelRequirements()
                                                            .getRoleType()));
         }
+        if (hearingDetails.getDuration() > HEARING_DETAILS_DURATION) {
```

Comment:
Optional: for readability/potential re-use could move this to `HearingDetails`
```suggestion
        if (hearingDetails.isMultiDayHearing()) {
```
where
```
public boolean isMultiDayHearing() {
    return getDuration() > HEARING_DETAILS_DURATION;
}
```